exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,33 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0) {
```

Comment:
Whenever you have this kind of logic: `if(expression) return true else return false` - you could simplify it by returning the expression directly: `return expression` - as the expression will then be evaluated to true or false anyway.